scripts/annotation.js

```diff
@@ -137,8 +138,27 @@ $(document).ready( ()=> {
             viewer.fullPageButton.removeAllHandlers();
             viewer.fullPageButton.addHandler("click", closeSeaDragon);
         }
+        else {
+            viewer.fullPageButton.addHandler("click", ()=> {
+                setTimeout(()=>{viewer.viewport.goHome(true)}, 100);
+            });
+        }
         viewer.open(tileSources);
         seadragon.show();
+        viewer.addHandler("full-screen", (eventData)=> {
+            fullScreenButton = $("[title|='Toggle full page']");
+            if (eventData.fullScreen) {
+                // changing to fullscreen
+                fullScreenButton.css("left", ()=> {
+                    return $('body').width() - 120;
```

Comment:
Yea it is a magic number...the button images are both 60px wide, so I have to subtract the width of both in order to get it on the right side.